frontend/src/main/utils/sortHelper.js

```diff
@@ -0,0 +1,55 @@
+
+// compareValues comes from this article
+//   https://www.sitepoint.com/sort-an-array-of-objects-in-javascript/
+// It is used to sort arrays of objects, like this:
+// For
+//  const singers = [
+//     { name: 'Steven Tyler', band: 'Aerosmith', born: 1948 },
+//     { name: 'Karen Carpenter', band: 'The Carpenters', born: 1950 },
+//     { name: 'Kurt Cobain', band: 'Nirvana', born: 1967 },
+//     { name: 'Stevie Nicks', band: 'Fleetwood Mac', born: 1948 },
+//   ];
+//
+// Use: 
+// // array is sorted by band, in ascending order by default
+// singers.sort(compareValues('band'));
+
+// // array is sorted by band in descending order
+// singers.sort(compareValues('band', 'desc'));
+
+// // array is sorted by name in ascending order
+// singers.sort(compareValues('name'));
+
+// // array is sorted by date if birth in descending order
+// singers.sort(compareValues('born', 'desc'));
+
+// Stryker disable next-line StringLiteral: The "" mutant results in equivalent mutant due to it defaulting to ascending sort if order is not "desc"
```

Comment:
Do we need these comments? I guess they are good for documentation purposes but may make it hard to read.

Reply:
I think these comments are probably ok... the first set is documenting how the module works; the Stryker comment is needed because this line will be an "equivalent mutant" that survives.

This is different from, say, "commented out code".